ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -48,6 +49,10 @@ interface PaginationArgs {
   limit?: number;
 }
 
+const headers = {
+  "User-Agent": `aptos-ts-sdk/${VERSION}`,
```

Comment:
Since we're using the same string in both cases, can you make a function in some `utils` file that returns this `aptos-ts-sdk/${VERSION}` string?

Reply:
Good call!